### PR TITLE
fix(outputs.yandex_cloud_monitoring): catch int64 values

### PR DESCRIPTION
**Pull request by powersj**
fixes: #12155

### CODE REVIEW

**Review by sspaink — APPROVED** [review]
Thanks for fixing this so quickly!

File: plugins/outputs/yandex_cloud_monitoring/yandex_cloud_monitoring.go [hunk 1]
```diff
@@ -117,18 +117,35 @@ func (a *YandexCloudMonitoring) Close() error {
...
+		return float64(-1), fmt.Errorf("expected float64, received %T: '%s'", value, value)
+	}
+}
+
 // Write writes metrics to the remote endpoint
 func (a *YandexCloudMonitoring) Write(metrics []telegraf.Metric) error {
 	var yandexCloudMonitoringMetrics []yandexCloudMonitoringMetric
 	for _, m := range metrics {
 		for _, field := range m.FieldList() {
+			value, err := setValue(field.Value)
```
Comment (srebhan) [inline plugins/outputs/yandex_cloud_monitoring/yandex_cloud_monitoring.go:136]: You might want to use `ToFloat64()` of the [internal package](https://github.com/influxdata/telegraf/blob/497297a92aa927ea4d0fffc2e88d10f7e6d87027/internal/type_conversions.go#L48), but not a must.

**Review by srebhan — APPROVED** [review]
Thanks for the quick fix @powersj! One suggestion, but not a must.

File: plugins/outputs/yandex_cloud_monitoring/yandex_cloud_monitoring.go (hunk 1 above)
Comment (powersj) [reply plugins/outputs/yandex_cloud_monitoring/yandex_cloud_monitoring.go:136]: oh hey look at that! I'll use that

**Review by MyaLongmire — APPROVED** [review]

**Pull request merged**